cli/src/main/java/com/box/l10n/mojito/cli/command/checks/CliCheckerOptions.java

```diff
@@ -40,23 +33,31 @@ public CliCheckerOptions(Set<PlaceholderRegularExpressions> parameterRegexSet, S
     }
 
     public String getDictionaryAdditionsFilePath() {
-        return dictionaryAdditionsFilePath;
+        return optionsMap.get(DICTIONARY_ADDITIONS_PATH_KEY);
     }
 
     public String getGlossaryFilePath() {
-        return glossaryFilePath;
+        return optionsMap.get(GLOSSARY_FILE_PATH_KEY);
     }
 
     public String getDictionaryFilePath() {
-        return dictionaryFilePath;
+        return optionsMap.get(DICTIONARY_FILE_PATH_KEY);
```

Comment:
Given that all of these values are now stored into a dict and there's wrapper methods to get all of the right values - would a strongly typed class with those properties be a better fit here in terms of validation?

That would make CheckerOptionsConverter a bit more involved - but it would also allow us to check that no bogus values are provided (e.g.: if someone misspells a key name) or no important keys are missing.